docs/src/get-started/tutorials/first-calculation.py

```diff
@@ -0,0 +1,210 @@
+# object with hyper parameters defined above.
+
+calculator = SphericalExpansion(**HYPER_PARAMETERS)
+descriptor0 = calculator.compute(frame0)
+print(type(descriptor0))
+
+# %%
+#
+# The descriptor format is a :class:`equistore.tensor.TensorMap` object.
+# Equistore is like Numpy for storing a representation of atomistic ML data.
+# Extensive details on the equistore are covered in the `corresponding
+# documentation <https://lab-cosmo.github.io/equistore/>`_.
+#
+# We will now have a look at how the data is stored inside
+# :class:`equistore.tensor.TensorMap` objects.
+
+
+print(descriptor0)
+
+# %%
+#
+# The :class:`equistore.tensor.TensorMap` is structured in ``blocks``, each
+# associated with a key. Here we have one block for each angular channel
+# labeled by ``spherical_harmonics_l``,
+# the central atom species ``species_center`` and neighbor atom species
+# labeled by ``species_neighbor``. Different atomic species are
+# represented using their atomic number, e.g. 1 for hydrogen, 6 for carbon, etc.
+# Our descriptor contains 112 blocks covering all combinations of the angular
+# channels and the elements.
+#
+# Let us take a look at the first block in detail. The 0th block contains the
+# descriptor for the 0th angular channel between the hydrogen-hydrogen pairs.
+
+print(descriptor0.keys[0])
+
+# %%
+#
+# The values are stored in an array. Let us take a look at its dimensions given by
+# the shape attribute.
+
+
+print(descriptor0.block(1).values.shape)
+
+# %%
+# The first dimension is 8 because we have eight hydrogen atoms in our frame.
+# The second dimension
+# is associated with the selected angular channel and has a size of
+# :math:$2l + 1$, where :math:`l` is the current
+# ``spherical_harmonics_l`` channel.
+# Here it dimension is 1 because we are looking at the ``spherical_harmonics_l=0``
+# channel. The last value represents the number
+# of radial channels. We choose ``max_radial=9`` in the hyper parameters above
+# so there are 9 values in the last dimension.
+#
+# Let's take a look at the values of the representation (also called features)
+# associated with the first hydrogen:
+
+print(descriptor0.block(0).values[0])
+
+# %%
+#
+# As you see the values are floating point numbers between -1 and 1. Values in
+# this range are reasonable and can be directly used as an input for a machine
+# learning algorithm.
+#
+# Rascaline is also able to process more than one structure within one function
+# call. You can process a whole dataset with
+
+descriptor_full = calculator.compute(frames)
+print(descriptor_full.block(0).values.shape)
+
+# %%
+#
+# Now, the 0th block of the :class:`equistore.tensor.TensorMap` contains not 8
+# but 420 entries in the first dimensions. This reflects the fact that in total
+# we have 420 hydrogen atoms in the whole dataset.
+#
+#
+# If you want to use another calculator as the
+# :class:`rascaline.calculators.SphericalExpansion` shown here
+# check out the :ref:`userdoc-references` section.
```

Comment:
```suggestion
# If you want to use another calculator instead of
# :class:`rascaline.calculators.SphericalExpansion` shown here
# check out the :ref:`userdoc-references` section.
```